javascript/javascript1/week4/VoiceAssistant.js

```diff
@@ -0,0 +1,74 @@
+const myObj = {
+    name: 0,
+    toDo: []
+}
+function getReply(command) {
+    if (command.split(' ').shift() === 'Set') {
+        let time = command.split('Set a timer for ').pop().split(' minutes')[0];
+        console.log(`Timer set for ${time} minutes`);
+        setTimeout(message, (time*60)*1000);
```

Comment:
nit 
```javascript
setTimeout(function() {
   console.log('Timer done')
}, time * 60 * 1000)
```
would also work. 
Overall choice between those options are more matter of personal preferences